sensor/common/registry/registry_store.go

```diff
@@ -12,7 +12,6 @@ import (
 	"github.com/stackrox/rox/pkg/expiringcache"
 	"github.com/stackrox/rox/pkg/logging"
 	"github.com/stackrox/rox/pkg/registries"
-	dockerFactory "github.com/stackrox/rox/pkg/registries/docker"
 	rhelFactory "github.com/stackrox/rox/pkg/registries/rhel"
 	"github.com/stackrox/rox/pkg/registries/types"
 	registryTypes "github.com/stackrox/rox/pkg/registries/types"
```

Comment:
is it me, or are these the same package?

Reply:
Those are indeed the same package. Interesting that our linters don't catch that.